robot/pmm/tests/browser/Service/new_service.robot

```diff
@@ -25,6 +28,7 @@ Setup Test Data
 
 *** Test Cases ***
 Create Service from top nav
```

Comment:
Add Documentation and feature:Service tag

Reply:
Done